eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+- job: Build
+  strategy:
+    matrix:
+      Linux:
+        vm.image: 'ubuntu-16.04'
+      Win:
+        vm.image: 'windows-2019'
+      MacOS:
+        vm.image: 'macOS-10.14'
+  pool:
+    vmImage: $(vm.image)
+  steps:
+  - task: CMake@1
```

Comment:
@bsiegel currently, it tries to build all projects by using `CMakeLists.txt` in the root. I will need to update it to is `$(ServiceDirectory)`.